curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba41a77db20a93f9d7a1.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 66c8ba41a77db20a93f9d7a1
+title: Step 1
+challengeType: 1
+dashedName: step-1
+---
+
+# --description--
+
+In this workshop, you'll learn JavaScript conditionals by building a loan qualification checker app.
+
+The app will check whether the user is eligible for a duplex, car, and condo loan.
```

Comment:
I think this should be expanded to say that the app checks for eligibility based on their income and credit score